### PR TITLE
Refactored others type of subotimal asserts

### PR DESCRIPTION
**Pull request by freddiewanah**
### Description

As discussed in PR #7609, I tried to break the suboptimal test refactors to smaller pieces. In this PR all asserts are checking textual content or instance of a parameter.

Suboptimal Assert: Instead of using statements such as assertIsNone, assertIsInstance, always simply use assertTrue or assertFalse. This will decrease the code overall readability and increase the execution time as extra logic needed.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: tests/test_get_package_version.py [hunk 1]
```diff
@@ -20,14 +20,14 @@ class TestGetVersion(unittest.TestCase):
 
     def test_default(self):
         output = get_package_version("42foobarnoexist")
-        self.assertTrue("UNKNOWN" in output)
+        self.assertIn("UNKNOWN", output)
 
         output = get_package_version("numpy")
-        self.assertFalse("UNKNOWN" in output)
+        self.assertIn("UNKNOWN", output)
```
Comment (KumoLiu) [inline tests/test_get_package_version.py:26]:
```suggestion
        self.assertNotIn("UNKNOWN", output)
```

File: tests/test_integration_bundle_run.py [hunk 2]
```diff
@@ -135,9 +135,9 @@ def test_scripts_fold(self):
         command_run = cmd + ["run", "training", "--config_file", config_file, "--meta_file", meta_file]
         completed_process = subprocess.run(command_run, check=True, capture_output=True, text=True)
         output = repr(completed_process.stdout).replace("\\n", "\n").replace("\\t", "\t")  # Get the captured output
-        print(output)
+        # print(output)
```
Comment (KumoLiu) [inline tests/test_integration_bundle_run.py:138]:
```suggestion
```

File: tests/test_integration_bundle_run.py [hunk 3]
```diff
@@ -149,8 +149,8 @@ def test_scripts_fold(self):
         ]
         completed_process = subprocess.run(command_run_workflow, check=True, capture_output=True, text=True)
         output = repr(completed_process.stdout).replace("\\n", "\n").replace("\\t", "\t")  # Get the captured output
-        print(output)
-        self.assertTrue(expected_condition in output)
+        # print(output)
```
Comment (KumoLiu) [inline tests/test_integration_bundle_run.py:152]:
```suggestion
```

**Review by KumoLiu — APPROVED** [review]
Thanks for the PR, overall looks good to me.

Only several minor comments inline.

**Review by atbenmurray — APPROVED** [review]
Looks good with @KumoLiu's suggestions

**Comment by freddiewanah** [comment]
Thanks for the review. All prints have been removed.

**Comment by KumoLiu** [comment]
Hi @freddiewanah, could you please fix the flake8 error, thanks.
https://github.com/Project-MONAI/MONAI/actions/runs/8801467478/job/24154880380?pr=7672#step:7:383

**Pull request merged**